src/lib/mana.ts

```diff
@@ -21,3 +22,22 @@ export function buildManaAuthorization(address: string, chainId: ChainId, contra
     chainId
   }
 }
+
+/**
+ * Get's value and tries to parse it with the supplied amount of decimals.
+ * It'll return the value as is if it's an invalid number or it doesn't have more than decimals than the upper limit.
+ */
+export function toFixedMANAValue(strValue: string, maximumFractionDigits = MAXIMUM_FRACTION_DIGITS): string {
```

Comment:
This implementation is repeated [in the marketplace PR](https://github.com/decentraland/marketplace/pull/620/files#diff-66bbe696b092faa35294af41eb315bdd25de307de34b4770a9b492a2d884aa66R36). Maybe we should move it to dapps?

Reply:
is it used in `decentraland-ui` (ie. for the balance in `Navbar` and `UserMenu`)? If so, I'd add it to dcl-ui rather than dcl-dapps i think